src/interpolations.jl

```diff
@@ -436,13 +445,12 @@ nfacets(ip::InterpolationByDim{2}) = nedges(ip)
 nfacets(ip::InterpolationByDim{1}) = nvertices(ip)
 
 """
-    is_discontinuous(::Interpolation)
-    is_discontinuous(::Type{<:Interpolation})
+    is_discontinuous(ip::Interpolation)
 
-Checks whether the interpolation is discontinuous (i.e. `DiscontinuousLagrange`)
+Checks whether the interpolation is discontinuous (i.e. `DiscontinuousLagrange`),
+automatically implemented based on `conformity(ip)`.
 """
-is_discontinuous(ip::Interpolation) = is_discontinuous(typeof(ip))
-is_discontinuous(::Type{<:Interpolation}) = false
+is_discontinuous(ip::Interpolation) = is_discontinuous(conformity(ip))
```

Comment:
Now that we have a better granularity to describe the continuity, I would argue that this function should be removed, as it is really just the check that `conformity !=  H1`.

Reply:
True, although we could also have e.g. H2, but for the vtk export that would probably be different anyways - I'll remove instead! 
Good to reduce some code sometimes 😄

Reply:
Yea, `is_discontinuous` was essentially the initial idea to distinguish the L2 space from H1 space without introducing the trait system (as at that time people argued against the trait system).